src/transport.cpp

```diff
@@ -1206,7 +1207,7 @@ namespace quicr {
         }
 
         auto result = quic_transport_->Enqueue(
-          track_handler.connection_handle_, track_handler.publish_data_ctx_id_, data, priority, ttl, 0, eflags);
+          track_handler.connection_handle_, track_handler.publish_data_ctx_id_, 0, data, priority, ttl, 0, eflags);
```

Comment:
`SendData()` is used with pipelining.  We need to talk about this a little more because pipelining is group unaware... Setting it to group 0 will result in not purging all related objects on expiry. Instead, it'll empty the whole queue of pipeline data.

Reply:
PR #605 resolves this.